sdk/docs/platform/README.md

```diff
@@ -1,14 +1,14 @@
 # Azure SDK Platform & Http Abstractions
 
-The Azure SDK platform layer provides abstractions for platform specific apis which are required by the Azure SDK.  The Azure SDK transport layer provides an abstraction for replacing the http stack.
+The Azure SDK platform layer provides abstractions for platform specific APIs which are required by the Azure SDK.  The Azure SDK transport layer provides an abstraction for replacing the HTTP stack.
 
 ## Platform
 
 Azure SDK Core depends on some system-specific functions. These functions are not part of the C99 standard library and their implementation depends on system architecture (for example a clock, thread sleep, or interlock).
 
 Azure SDK provides three platform implementations for you, one for Windows (`az_win32`), another for Linux and MacOS (`az_posix`) and an empty implementation (`az_noplatform`).
 
-See [compiler options](https://github.com/Azure/azure-sdk-for-c#compiler-options) to learn about how to build and use each of these available targets or how to use your own platform implementation.
+See [compiler options][azure_sdk_compiler_options] to learn about how to build and use each of these available targets or how to use your own platform implementation.
```

Comment:
The headline is called `cmake options`, not `compiler options` in the README docs. We should change the name and the link. Right now the link doesn't point anywhere.
https://github.com/Azure/azure-sdk-for-c/blob/master/README.md#compiler-options
```suggestion
See [cmake options][azure_sdk_cmake_options] to learn about how to build and use each of these available targets or how to use your own platform implementation.
```